src/Step/SkipStep.php

```diff
@@ -39,9 +39,14 @@ public function __construct(LoggerInterface $logger, string $stepName)
      * Logs the information about step skipping.
      *
      * @return void
+     * @throws StepException
      */
     public function execute()
     {
-        $this->logger->info(sprintf('Step "%s" was skipped', $this->stepName));
+        try {
```

Comment:
Why did you add try/catch here? Logger method `info` doesn't throw any exceptions.

Reply:
changes reverted